user_form_wrapper.html

```diff
@@ -176,7 +176,8 @@
     function showMobileFields() {
       // add mobile field if not already added, or show if already added
       if (!jQuery('.mobile_phone_box').length) {
-        jQuery('#known_or_unknow_fields').append(jQuery('<div id="ak-fieldbox-mobile_phone" class="input-text ak-input-type-user  ak-err-below"><label for="id_mobile_phone">Mobile phone</label><input type="text" name="mobile_phone" id="id_mobile_phone" class="ak-userfield-input"></div>'));
+
+        jQuery('#known_or_unknow_fields').append(jQuery('<div id="ak-fieldbox-mobile_phone" class="input-text ak-input-type-user mobile_phone_box ak-err-below"><label for="id_mobile_phone">Mobile phone</label><input type="text" name="mobile_phone" id="id_mobile_phone" class="ak-userfield-input"></div>'));

```

Comment:
Is `unknow` correct here in ` jQuery('#known_or_unknow_fields')`?

Reply:
It looks like the only change here is removing `mobile_phone_box`, is that correct? If so, are we sure that won't have ill effects on other pages?

Reply:
yeah i don't want to fix the typo in case it breaks something else haha